lib/server/noded.py

```diff
@@ -710,6 +710,15 @@ def perspective_instance_migrate(params):
     return backend.MigrateInstance(cluster_name, instance, target, live)
 
   @staticmethod
+  def perspective_instance_start_postcopy(params):
+    """ Moves a migrating instance from precopy to postcopy mode
```

Comment:
Nit: To keep a somewhat consistent docstring, use either "move" or "switch" both here and in the rpc_defs.py file.